api/tests/Feature/PoolCandidateUpdateTest.php

```diff
@@ -1229,4 +1229,143 @@ public function testManualStatusUpdatePolicy(): void
+        $this->poolCandidate->placed_at = $past;
+        $this->poolCandidate->save();
+
+        $this->actingAs($this->$user, 'api')
+            ->graphQL($this->manualStatusUpdateMutation, [
+                'id' => $this->poolCandidate->id,
+                'candidate' => ['status' => $status],
+            ])->assertJsonFragment([
+                'status' => [
+                    'value' => $status,
+                ],
+                ...$expected,
+            ]);
+    }
+
+    public static function nullTimeProvider()
+    {
+        $nullState = [
+            'removedAt' => null,
+            'finalDecisionAt' => null,
+            'placedAt' => null,
+        ];
+
+        $finalDeciscion = [
+            'removedAt' => null,
+            'placedAt' => null,
+        ];
+        $removed = [
+            'finalDecisionAt' => null,
+            'placedAt' => null,
+        ];
+        $placed = [
+            'removedAt' => null,
+        ];
+
+        return [
+            'new application' => [
+                PoolCandidateStatus::NEW_APPLICATION->name,
+                $nullState,
+                'communityAdminUser',
+            ],
+            'application review' => [
+                PoolCandidateStatus::APPLICATION_REVIEW->name,
+                $nullState,
+                'communityAdminUser',
+            ],
+            'screened in' => [
+                PoolCandidateStatus::SCREENED_IN->name,
+                $nullState,
+                'communityAdminUser',
+            ],
+            'under assessment' => [
+                PoolCandidateStatus::UNDER_ASSESSMENT->name,
+                $nullState,
+                'communityAdminUser',
+            ],
+            'screened out - application' => [
+                PoolCandidateStatus::SCREENED_OUT_APPLICATION->name,
+                $finalDeciscion,
+                'communityAdminUser',
+            ],
+            'screened out - not interested' => [
+                PoolCandidateStatus::SCREENED_OUT_NOT_INTERESTED->name,
+                $removed,
+                'communityAdminUser',
+            ],
+            'screened out - not responsive' => [
+                PoolCandidateStatus::SCREENED_OUT_NOT_RESPONSIVE->name,
+                $removed,
+                'communityAdminUser',
+            ],
+            'screened out - assessment' => [
+                PoolCandidateStatus::SCREENED_OUT_ASSESSMENT->name,
+                $finalDeciscion,
+                'communityAdminUser',
+            ],
+            'qualified - available' => [
+                PoolCandidateStatus::QUALIFIED_AVAILABLE->name,
+                $finalDeciscion,
+                'communityAdminUser',
+            ],
+            'qualified - available' => [
+                PoolCandidateStatus::QUALIFIED_AVAILABLE->name,
+                $finalDeciscion,
+                'communityAdminUser',
+            ],
+            // 'qualified - unavailable' => [
```

Comment:
It seems like these run OK, right?  They're just not giving you the result you expect?  It's not a user permission problem?

Reply:
Hmm, I was actually getting an unauthorized error on them :thinking:

Reply:
Okay, not sure what was going on but they do work as expected. Added back  in [fix tests](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12734/commits/85a495b5af47cef5016356d9676659f6d599289f)

Reply:
That one actuall does seem to be impossible to set.. No matter what I get unauthorized both in the tests and UI. Now, I have no idea if that is expected or not so it may be another bug :woman_shrugging: 

![2025-02-10_15-14](https://github.com/user-attachments/assets/000be42c-a2a2-4547-a40c-8d503bb559a3)

Reply:
Alright, sounds good. [add expired back with comment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12734/commits/ce7f0fcdd2b09403cf3d8402c761dfe99454b9e1)